bulk_grades/views.py

```diff
@@ -15,37 +15,28 @@
 log = logging.getLogger(__name__)
 
 
-class GradeOnlyExportMixin(object):
+class GradeOnlyExport(View):
     """
     CSV Export of grade information only. To be used by both bulk grade export and interventions.
     """
 
+    def __init__(self, **kwargs):
+        """
+        Configure initial state.
+        """
+        super(GradeOnlyExport, self).__init__(**kwargs)
+        self.processor = None
+        self.operation_id = ''
```

Comment:
Strictly speaking, since this class is for export, `operation_id` will never be used.